src/sentry/auth/helper.py

```diff
@@ -379,6 +382,19 @@ def _post_login_redirect(self) -> HttpResponseRedirect:
 
         return response
 
+    def _check_verification_key(self, identity):
```

Comment:
nit: I like methods that return a boolean to have a semantic name that answers a yes/no question. for example, `_has_verified_account`